src/core/core.controller.js

```diff
@@ -961,6 +967,11 @@ class Chart {
 		});
 	}
 
+	/**
+	 * @param {{datasetIndex: number, index: number, element: Element}[]} items
```

Comment:
I would bet that `{datasetIndex: number, index: number, element: Element}` is used throughout `interactions.js`, so it might be good to make it a `typedef` there and then import it here rather than duplicating